src/Activator.php

```diff
@@ -4,6 +4,8 @@
 class Activator {
 	public function __construct( $file ) {
 		register_activation_hook( $file, [ $this, 'activate' ] );
+
+		add_filter( 'plugin_action_links_' . plugin_basename(SLIM_SEO_FILE), [ $this, 'add_settings_link' ] );
```

Comment:
Can you add some spaces to address the WP coding standards?

Reply:
I hope I got it now